criu/bpfmap.c

```diff
@@ -1,6 +1,7 @@
 #include <stdio.h>
 #include <linux/bpf.h>
 #include <bpf/bpf.h>
+#include <bpf/libbpf_common.h>
```

Comment:
It seems this is redundant -- `bpf.h` explicitly includes `libbpf_common.h` (checked with v0.4 and v0.7)